Sources/Array+Remove.swift

```diff
@@ -35,7 +35,7 @@ extension Array {
 
      - parameter object: The object to be removed (if found)
      */
-    mutating func removeObject<T: Equatable>(_ object: T) {
+    public mutating func removeObject<T: Equatable>(_ object: T) {
```

Comment:
Nice, thank you. I took a look and will incorporate that change now if that's ok